.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,11 +2,13 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: 'bug'
-assignees: ''
+labels: "bug \U0001F41E, help wanted"
```

Comment:
I did this from the issue editing template. I think that's a unicode emoji someone added to the label?